dpnp/dpnp_array.py

```diff
@@ -1050,7 +1050,47 @@ def put(self, indices, vals, /, *, axis=None, mode="wrap"):
 
         return dpnp.put(self, indices, vals, axis=axis, mode=mode)
 
-    # 'ravel',
+    def ravel(self, order="C"):
+        """
+        Return a contiguous flattened array.
+
+        For full documentation refer to :obj:`numpy.ndarray.ravel`.
```

Comment:
I am not sure what is the general practice here. 
Maybe we can add "Refer to :obj:`dpnp.ravel` for full documentation." And remove the repetitive explanation.